web/src/emoji_picker.js

```diff
@@ -282,6 +282,12 @@ function filter_emojis() {
 }
 
 function toggle_reaction(emoji_name, event) {
+    // The emoji picker for setting user status
+    // doesn't have a concept of toggling.
+    if ($("#set-user-status-modal").length) {
+        return;
+    }
```

Comment:
Opened https://github.com/zulip/zulip/issues/28464 as a follow-up to this.

Reply:
Let’s reference that in some kind of TODO comment here.